docs/openapi/definitions/auth.yaml

```diff
@@ -0,0 +1,217 @@
+openapi: 3.0.1
+info:
+  title: Auth API
+  description:
+    Backstage passport-based auth-provider API. Each provider that is configured
+    in `@backstage/auth-backend` is supplied with the documented endpoints.
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.1.1-alpha.7
+externalDocs:
+  description: Backstage official documentation
+  url: https://github.com/spotify/backstage/blob/master/docs/README.md
+servers:
+  - url: http://localhost:7000/auth/google
+tags:
+  - name: provider
+    description: List of endpoints per provider
+paths:
+  /start:
+    get:
+      tags:
+        - provider
+      summary:
+        Initializes the authorization flow and redirects to a consent popup for
+        the provider
+      description:
+        The popup window is pointed to the `/start` endpoint of the auth
+        provider in the auth-backend plugin, which then redirects to the OAuth
+        consent screen of the provider.
+      parameters:
+        - name: scope
+          in: query
+          description: Scope requested for the auth session
+          required: true
+          schema:
+            type: string
+            example: email
+        - name: env
+          in: query
+          description: Parameter to indicate runtime environment
+          required: true
+          schema:
+            type: string
+            example: development
+      responses:
+        302:
+          description: Redirect to the consent popup
+          headers:
+            Location:
+              description: The url to the consent popup
+              schema:
+                type: string
+            Set-Cookie:
+              description: Nonce cookie
+              schema:
+                $ref: '#/components/schemas/Nonce-Cookie'
+        default:
+          description: An error occurred
+  /frame/handler:
+    get:
+      tags:
+        - provider
+      summary: Handles the response from the consent popup for the provider
+      description:
+        If the login request is accepted, the popup window will be redirected
+        back to the `/handler/frame` endpoint of the auth backend.
+      parameters:
+        - name: scope
+          in: query
+          description: Scope requested for the auth session
+          required: true
+          schema:
+            type: string
+            example: email
+        - name: code
+          in: query
+          description:
+            Short-term authorization code used to be exchanged for long-term
```

Comment:
```suggestion
            Short-term authorization code used to be exchanged for access
```

Reply:
let's not say long-term since access tokens are short-lived in some cases and in some cases not (Github)